home.html

```diff
@@ -0,0 +1,132 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Flawless Shop</title>
+  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" />
+</head>
+<body>
+  <header>
+    <nav class="nav">
+      <div class="nav-icons">
+        <ul>
+         <li><i class="fa fa-twitter"></i></li> 
+         <li><i class="fa fa-instagram"></i></li> 
+         <li><a href="">French</a></li>
+         <li><a href="">My page</a></li>
+         <li><a href="">Logout</a></li>
+        </ul>
+      </div>
+      <div class="nav-bar">
+        <a href="home.html"><img class="click" src="images/logo.png"></a>
```

Comment:
please kindly add `alt ` attribute for all your image property as it shows good html semantics 😄